src/nvidia_rag/utils/image_utils.py

```
@@ -0,0 +1,44 @@
+                return image_url
+        else:
+            b64_data = image_url
+
+        image_bytes = base64.b64decode(b64_data)
+        img = PILImage.open(io.BytesIO(image_bytes)).convert("RGB")
+
+        with io.BytesIO() as buffer:
+            img.save(buffer, format="PNG")
+            return base64.b64encode(buffer.getvalue()).decode("utf-8")
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)